packages/fxa-settings/src/components/PageAvatar/index.tsx

```diff
@@ -48,6 +54,7 @@ export const PageAddAvatar = (_: RouteComponentProps) => {
 
   const uploadAvatar = useAvatarUploader({
     onSuccess: (newAvatar) => {
+      logViewEvent(settingsViewName, 'avatar.crop.submit.change');
```

Comment:
nfw im doing all the mocking to assert this mf

Reply:
this is exactly why I opened #7596